.github/workflows/main.yml

```diff
@@ -13,6 +13,12 @@ jobs:
       - uses: "actions/setup-python@v2"
         with:
           python-version: "3.9"
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
```

Comment:
This should probably be hashing both `requirements.prod.txt` and `requirements.dev.txt` (it's not hashing anything atm!).

Reply:
doh! you're right - it was using the fallback key

Reply:
I missed that -- how embarassing. Sorry!

Reply:
I just did exactly the same thing in another repo, only reason I caught it!